packages/dcos-integration-test/buildinfo.json

```diff
@@ -2,6 +2,7 @@
   "requires": [
       "dcos-image-deps",
       "dcos-test-utils",
+      "prometheus_client",
```

Comment:
For clarity and consistency with other packages that provide Python libs, let's call this `python-prometheus_client`.